### ISSUE TITLE
payara openapi server information is wrong

### ISSUE THREAD

**Issue by marmots** [payara/Payara#3953]
# Description #
----------
Using payara server & micro at version 5.191. openapi server information in generated yaml file is wrong (protocols are changed)

in payara server:
http://localhost:8080/openapi

returns
...
servers:
- url: http://localhost:8181/benchmarks
  description: Default Server.
- url: https://localhost:8080/benchmarks
  description: Default Server.
...

in payara micro
http://localhost:8081/openapi

returns
...
servers:
- url: https://localhost:8081/
  description: Default Server.
...

## Expected Outcome
it seems that for some reason generated endpoints have mixed protocols and ports.

## Environment ##

- **Payara Version**: 4.1.1.x
- **Edition**: 
- **JDK Version**: 
- **Operating System**: 
- **Database**:

**Comment by jbee** [comment]
I think a similar bug has been reported. Have to look it up tomorrow.

**Comment by jbee** [comment]
Could not find it so far. Will see if I can reproduce the defect in current version.

**Comment by jbee** [comment]
I cannot reproduce the issue on master (5.192.RC5).

@marmots Can you please provide steps to reproduce? 

What I tried is deploying an application with some Open API relevant components and looking at http://localhost:8080/openapi which gives me
```yaml
openapi: 3.0.0
info:
  title: Deployed Resources
  version: 1.0.0
servers:
- url: http://x1:8080/testopenapi
  description: Default Server.
- url: https://x1:8181/testopenapi
  description: Default Server.
paths: ...
```
The problem might be solved or it is the way I am testing.

**Comment by jbee** [comment]
Tested with 5.191 (release version) which shows the reported problem:
```yaml
openapi: 3.0.0
info:
  title: Deployed Resources
  version: 1.0.0
servers:
- url: http://x1:8181/testopenapi
  description: Default Server.
- url: https://x1:8080/testopenapi
  description: Default Server.
```
So the issue has been fixed in 5.192 but I could not find the related change. As said in an earlier comment there was a change related to a HTTP/HTTPS confusion which I would assume caused this as well.

**Comment by jbee** [comment]
Finally found it: Fixed by #3916
